utils/DateUtils.js

```diff
@@ -53,6 +53,16 @@ class DateUtils {
         return imageDate.isSame(date);
     }
 
+    static isDateOverFourWeeks(date) {
+        const dateFourWeeksLater = moment().add(4, 'weeks');
+        return date.isSameOrAfter(dateFourWeeksLater);
+    }
```

Comment:
This should be renamed to isDateGreatThanOrEqualToFourWeeks as you're using isSameOrAfter.